README.rst

```diff
@@ -29,9 +29,10 @@ common case:
 Client side Requirements
 ------------------------
 
-- sudo, or logged in as root on your client machine.
+- sudo, or root access on your client machine.
   (The server doesn't need admin access.)
 - Python 2.7 or Python 3.5.
+- For Python 2.7, you need PyXAPI, available `here <http://www.pps.univ-paris-diderot.fr/~ylg/PyXAPI/>`_
```

Comment:
PyXAPI is only required for Python 2.7 *and* TProxy with UDP or DNS enabled. That is why it was below the UDP or DNS paragraph. I realize this was not clear however... Maybe it should be included in that paragraph.